src/nvim/os/os_defs.h

```diff
@@ -25,6 +25,10 @@
 # define MAXNAMLEN NAME_MAX         /* for Linux before .99p3 */
 #endif
 
+#if defined(_MAX_PATH) && !defined(MAXNAMLEN)
+# define MAXNAMLEN _MAX_PATH       /* for Windows */
```

Comment:
Because the other ones (`MAXNAMLEN`) for Unix were in this file.

We could of course move that stuff. I just left it here to be consistent.